### PR TITLE
add Hydras & Co. book to documentation

### PR DESCRIPTION
**Pull request by palmskog**
cc: @Zimmi48 @Casteran

### CODE REVIEW

File: pages/documentation.html [hunk 1]
```diff
@@ -106,6 +106,11 @@
 with the SSReflect proof language, and the Mathematical Components
 library.</li>
 
+<li><a href="">Hydras &amp; Co.</a> (Pierre Castéran <em>et al.</em>, 2021),
```
Comment (Zimmi48) [inline pages/documentation.html:109]: I think the following suggestion could be a way of insisting on the continuously in-progress aspect.

```suggestion
<li><a href="">Hydras &amp; Co.</a> (Pierre Castéran <em>et al.</em>, 2021–⁠),
```

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
By the way, the URL needs to be filled in.

File: pages/documentation.html (hunk 1 above)
Comment (palmskog) [reply pages/documentation.html]: How about "2021-ongoing"?

Comment (Zimmi48) [reply pages/documentation.html]: Note that I was using an en-dash. Your call on the "ongoing" addition, but that's not something that I've seen anywhere before.

Comment (palmskog) [reply pages/documentation.html]: "2021–⁠present" then?

Comment (Zimmi48) [reply pages/documentation.html]: Sounds better to me.

**Pull request merged**